migration-guide/src/explorer-api.md

```diff
@@ -0,0 +1,2904 @@
+
+Get summary information about an address.
+
+=== "cardano-rest"
+
+    ```console
+    $ https://explorer.cardano-testnet.iohkdev.io/api/addresses/summary/addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5
+    ```
+
+    <details>
+      <summary>see JSON response</summary>
+    ```json
+    {
+      "Right": {
+        "caAddress": "addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5",
+        "caType": "CPubKeyAddress",
+        "caChainTip": {
+          "ctBlockNo": 2225942,
+          "ctSlotNo": 16238640,
+          "ctBlockHash": "8c892cd82dcb7265e0714346961afddc6d2334a8cd46e0607bd512046ac1db0e"
+        },
+        "caTxNum": 1,
+        "caBalance": {
+          "getCoin": "590707773"
+        },
+        "caTotalInput": {
+          "getCoin": "590707773"
+        },
+        "caTotalOutput": {
+          "getCoin": "0"
+        },
+        "caTotalFee": {
+          "getCoin": "172101"
+        },
+        "caTxList": [
+          {
+            "ctbId": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+            "ctbTimeIssued": 1610575838,
+            "ctbInputs": [
+              {
+                "ctaAddress": "addr_test1qp6dnz3gs84zn7cjx3mrhquj99zfd0acqh9qjldeec9rp4zkvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqs553m2",
+                "ctaAmount": {
+                  "getCoin": "591879874"
+                },
+                "ctaTxHash": "984bf469edee7bcd19ddcf9b7c5cc7e107e10df5061d58dff99d81bb29466094",
+                "ctaTxIndex": 1
+              }
+            ],
+            "ctbOutputs": [
+              {
+                "ctaAddress": "addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5",
+                "ctaAmount": {
+                  "getCoin": "590707773"
+                },
+                "ctaTxHash": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+                "ctaTxIndex": 0
+              },
+              {
+                "ctaAddress": "addr_test1qrlh7p9th5c9ps938ry05vq96j92lzuhqr29v46caydf2wzkvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqt48df3",
+                "ctaAmount": {
+                  "getCoin": "1000000"
+                },
+                "ctaTxHash": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+                "ctaTxIndex": 1
+              }
+            ],
+            "ctbInputSum": {
+              "getCoin": "591879874"
+            },
+            "ctbOutputSum": {
+              "getCoin": "591707773"
+            },
+            "ctbFees": {
+              "getCoin": "172101"
+            }
+          }
+        ]
+      }
+    }
+    ```
+    </details>
+
+=== "cardano-graphql"
+
+    !!! info
+        The following fields from the REST endpoint are not supported: `caType`, `caChainTip`.
```

Comment:
TODO:
- caTotalFee -- this should be possible
- caType -- no
- caChainTip -- ??